### ISSUE TITLE
suggetion

### ISSUE THREAD

**Issue by Amrapalivishwa** [janastu/alipi#5]
for those who can not read English but can understand English (like me i cannot read Kannada but understand it) why not plug in Eng screen reader to alipi and sweet maker?
